lib/tasks/buildThemes.js

```diff
@@ -33,8 +33,10 @@ function getPool(taskUtil) {
 	return pool;
 }
 
-async function buildThemeInWorker(options, taskUtil) {
-	return getPool(taskUtil).exec("execThemeBuild", [options]);
+async function buildThemeInWorker(taskUtil, options, transferList) {
+	const toTransfer = transferList ? {transfer: transferList} : undefined;
+
+	return getPool(taskUtil).exec("execThemeBuild", [options], toTransfer);
```

Comment:
I wonder why the last argument hasn't been used in https://github.com/SAP/ui5-builder/pull/914. Does that mean that it never really used transferable objects in a correct way? I also just looked up https://github.com/josdejong/workerpool/blob/master/examples/transferableObjects.js / https://github.com/josdejong/workerpool/blob/master/examples/workers/transferableObjectsWorker.js which is different to the current approach of transferring objects.

Reply:
AFAICS PR #914 is using the "copy" approach, as defined in https://github.com/josdejong/workerpool/blob/8219f7deaf535310cb8c3f273cda8e047959502c/examples/transferableObjects.js#L24-L29.

Reply:
As we discussed, transferring objects wouldn't add benefits in terms of performance, but would make the code more complex. Leaving the copies for this implementation